app/models/data_repo_client.rb

```diff
@@ -92,6 +97,47 @@ def process_api_request(http_method, path, payload: nil, retry_count: 0)
   # API ENDPOINT BINDINGS
   ##
 
+  ##
+  # Status
+  ##
+
+  # get more detailed status information about TDR API
+  # this method doesn't use process_api_request as we want to preserve error states rather than catch and suppress them
+  #
+  # * *return*
+  #   - +Hash+ with health status information for various TDR services or error response
+  def api_status
+    path = self.api_root + '/status'
+    # make sure access token is still valid
+    headers = {
+      'Authorization' => "Bearer #{self.valid_access_token['access_token']}",
+      'Accept' => 'application/json',
+      'Content-Type' => 'application/json',
+      'x-app-id' => "single-cell-portal",
+      'x-domain-id' => "#{ENV['HOSTNAME']}"
+    }
+    begin
+      response = RestClient::Request.execute(method: :get, url: path, headers: headers)
+      JSON.parse(response.body)
+    rescue RestClient::ExceptionWithResponse => e
+      Rails.logger.error "Terra Data Repo status error: #{e.message}"
```

Comment:
```suggestion
      logger.error "Terra Data Repo status error: #{e.message}"
```

You refactored `Rails.logger` to `logger` in `search_controller.rb `, might as well do so in this file too.

Reply:
It actually is not - any controller automatically sets `logger` to `Rails.logger`, but other classes do not.

Reply:
Oof, I'll try to remember that.  Thanks for the note!